### ISSUE TITLE
Microsoft C++ exception: torch::jit::ErrorReport at memory location 0x0000001C0C3BBB40.

### ISSUE THREAD

**Issue by omeryasar** [pytorch/pytorch#48082]
Hi,
I try to work with torchvision detection models in c++ to be able to do that I obtain TorchScript models via torch.jit.script of fasterrcnn_resnet50_fpn and retinanet_resnet50_fpn but while I trying to load scripted models in C++ I get "Microsoft C++ exception: torch::jit::ErrorReport at memory location 0x0000001C0C3BBB40." exception which is very uninformative. I use latest versions of both torch, torchvision and libtorch. I wonder if there is anyone try same operations and succeeded if so any recommendation is appreciated. Thanks

**Comment by peterjc123** [comment]
Please use a try catch to wrap your code like the following style and share the results.
```cpp
    try
    {
       // your code goes here...
    }
    catch (std::exception &e)
    {
        std::cout << e.what() << std::endl;
    }
```

**Comment by omeryasar** [comment]
Hi , I get torch.cpu.pdb not loaded

**Comment by peterjc123** [comment]
If you are using the Release version of LibTorch, then we don't provide the PDBs. Otherwise, please copy the PDBs from `[LIBTORCH ROOT]\lib` to the directory of your executable.

**Comment by omeryasar** [comment]
Yeah it solves that one but it throws "not found schema_matching.cpp" now

**Comment by peterjc123** [comment]
@omeryasar You'll need a clone of PyTorch with the same version of LibTorch. Assume you use PyTorch v1.7.0, you may do that by doing `git clone https://github.com/pytorch/pytorch.git -b v1.7.0 --single-branch`.

**Comment by omeryasar** [comment]
@peterjc123 Thanks for your instant replies but I don't understand what should I do with cloned PyTorch afterwards could you please explain it

**Comment by omeryasar** [comment]
@peterjc123 Somehow I manage to compile it and get the exception message ; 
"schemas.size() > 0 INTERNAL ASSERT FAILED at "..\\..\\torch\\csrc\\jit\\frontend\\schema_matching.cpp":500, please report a bug to PyTorch.                                                                                                     Exception raised from matchSchemas at ..\..\torch\csrc\jit\frontend\schema_matching.cpp:500 (most recent call first):   00007FF8D88EF84A00007FF8D88EEBE0 c10.dll!c10::detail::LogAPIUsageFakeReturn [<unknown file> @ <unknown line number>]    00007FF8D88EF4BA00007FF8D88EEBE0 c10.dll!c10::detail::LogAPIUsageFakeReturn [<unknown file> @ <unknown line number>]    00007FF8D88F023100007FF8D88EEBE0 c10.dll!c10::detail::LogAPIUsageFakeReturn [<unknown file> @ <unknown line number>]"

**Comment by peterjc123** [comment]
Seems like a duplicate of https://github.com/pytorch/pytorch/issues/35881.

**Comment by omeryasar** [comment]
@peterjc123 And the issue you mentioned is still open is it dead end ?

**Comment by peterjc123** [comment]
@omeryasar Some one will take a look later. The original issue doesn't have the `windows` tag.

**Comment by dc986** [comment]
I have a similar problem. On Windows, VisualStudio 2019 with
libtorch 1.7.1+cpu
torchvision 0.8.2+cpu
I am trying to load fasterrcnn_resnet50_fpn but I got the exception
`Unhandled exception at 0x000007FEFCC0A06D in Test_Pytorch.exe: Microsoft C++ exception: torch::jit::ErrorReport at memory location 0x0000000000127440.`

I am loading the model with this code:
`torch::jit::script::Module module;
module = torch::jit::load(model_path);`

The model has been scripted with the same version of torch and torchvision.

```
model = torchvision.models.detection.fasterrcnn_resnet50_fpn(pretrained=False)
model.eval()

scripted_model = torch.jit.script(model)
scripted_model.save("my_scripted_model_save_torch171_tochvision082_false.pt")
```

Do you have suggestion?
Thanks

**Comment by Blackhex** [comment]
@mantaionut Can you please verify this is really the duplicate of [#35881](https://github.com/pytorch/pytorch/issues/35881) and if so remove the `module: windows` label?

**Comment by mantaionut** [comment]
I tested on linux both the fasterrcnn_resnet50_fpn and retinanet_resnet50_fpn and they fail just like on #35881 with ```
Unknown builtin op: torchvision::nms.
Could not find any similar ops to torchvision::nms. This op may not exist or may not be currently supported in TorchScript.
:
  File "code/__torch__/torchvision/ops/boxes.py", line 128
  _55 = __torch__.torchvision.extension._assert_has_ops
  _56 = _55()
  _57 = ops.torchvision.nms(boxes, scores, iou_threshold)
        ~~~~~~~~~~~~~~~~~~~ <--- HERE
  return _57 ```. 
So I will remove the windows label.

**Comment by lzy473** [comment]
我在wenet自己训练语音模型怎么实现web交互

**Comment by iremyux** [comment]
Since the Windows label was removed, I am removing it from 'Pytorch on Windows' project as well.

**Comment by janeyx99** [comment]
TorchScript has been deprecated in favor of torch.export as of 2.11. https://pytorch.org/blog/pytorch-2-11-release-blog/ Closing this issue as we are not further developing on it--please ping if you believe this issue was erroneously closed.
